doc/guides/dts/macros.bnf

```diff
@@ -50,6 +50,9 @@ node-macro =/ %s"DT_N" path-id %s"_PARENT"
 node-macro =/ %s"DT_N" path-id %s"_FOREACH_CHILD"
 ; The node's status macro
 node-macro =/ %s"DT_N" path-id %s"_STATUS_" dt-name
+; The node's dependency ordinal. This is a non-negative integer
+; value that is used to represent dependency information.
+node-macro =/ %s"DT_N" path-id %s"_DEP_ORD"
```

Comment:
My original was just `_ORD` because at this time we have only one ordinal associated with each node, which happens to represent one type of dependency information.

If we're adding `DEP` to identify the ordinal type I would prefer to have it be `_ORD_DEP`, reflecting a path from general to specific.  E.g. if we used the dependency ordinal for device power management, but needed a stronger (total) order for device initialization (e.g. incorporating init level), we'd add `_ORD_INIT`.  This also reduces the likelihood of namespace collision.

Just an outline of where my thoughts are right now, no change requested at this time.